File: libs/esl/perl/events.pl
require ESL;

my $con = new ESL::ESLconnection("localhost", "8021", "ClueCon");

$con->events("plain", "all");

while($con->connected()) {
  #my $e = $con->recvEventTimed(100);
  my $e = $con->recvEvent();

  if ($e) {
    #print $e->serialize();
    my $h = $e->firstHeader();
    while ($h) {
      printf "Header: [%s] = [%s]\n", $h, $e->getHeader($h);
      $h = $e->nextHeader();
    }

  }

}

File: libs/esl/python/events.py
#!/usr/bin/env python

import string
import sys

from ESL import *

con = ESLconnection("localhost","8021","ClueCon")
#are we connected?

if con.connected:

  con.events("plain", "all");

  while 1:
  #my $e = $con->recvEventTimed(100);
    e = con.recvEvent()

    if e:
      print e.serialize()
